### PR TITLE
All Pages

### PR DESCRIPTION
**Pull request by mimansha-swarup**
- wishlist Page
- Cart page
- signup Page
- LogIn Page

https://user-images.githubusercontent.com/69811810/160174219-fc72f24c-ed46-4b27-9a2f-a7644f0a6527.mp4

### CODE REVIEW

File: src/Context/categoriesContext.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export const CategoriesProvider = ({ children }) => {
+  const [categoriesData, setCategoriesData] = useState([]);
+  const [status, setStatus] = useState({ isLoading: false, wrror: "" });
+
+  useEffect(() => {
+    (async () => {
+      setStatus({ isLoading: true, error: "" });
+      const response = await axios.get(categoriesApi);
+      if (response.status === 200) {
+        // console.log(response.data)
```
Comment (manojasarsa) [inline src/Context/categoriesContext.js]: you can remove console.log if no longer needed

**Review by manojasarsa — APPROVED** [review]
Rest looks good to merge!

**Review by manojkumar8400 — APPROVED** [review]
UI is looking good, and the code is readable.

File: src/Context/categoriesContext.js (hunk 1 above)
Comment (mimansha-swarup) [reply src/Context/categoriesContext.js]: haa thanks for reminding

**Pull request merged**